remove proper noun Ethereum from all locales for onlyAddTrustedNetworks

brad-decker opened the pull request:
Fixes: #10388

Explanation:  
Removes proper noun 'Ethereum' from the translation strings. Going to need help verifying all locales. Note that I did not remove anything from the `ko` translation because the word `Ethereum` did not appear in the string.

darkwing reviewed (dismissed):
LGTM!

Gudahtt commented on app/_locales/zh_CN/messages.json:
```diff
@@ -1128,7 +1128,7 @@
   },
   "onlyAddTrustedNetworks": {
-    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
+    "message": "恶意的 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
```
When testing this in Google Translate, "Ethereum" still appears in the output :thinking:

brad-decker replied:
I'm not getting the same results, 
![image](https://user-images.githubusercontent.com/4448075/110352940-1e31d300-7ffc-11eb-8b74-d37733a5f82d.png)

brad-decker replied:
malicious blockchains, tho.

Gudahtt replied:
I think you're translating a different locale here (this is simplified Chinese, not Japanese)

brad-decker replied:
Oops, i was on the wrong locale.

Gudahtt reviewed (commented):
We might want to consider removing some or all of these translations. We might be making them confusing/grammatically incorrect by removing this term.

Also some of them still seem to include _translations_ of Ethereum. The `ko` message still includes 'Ethereum' in the output when translated with Google Translate for example.

brad-decker commented:
>We might want to consider removing some or all of these translations. We might be making them confusing/grammatically incorrect by removing this term.

Should we just go the safest route and remove them, waiting for lionbridge to update them in the future? cc @Gudahtt

Gudahtt commented:
That's essentially what we've done in most other cases where the _meaning_ of the message has changed. Deleting them seems appropriate. If there were any where you felt reasonable confident the sentences till makes sense without 'Ethereum' we could make exceptions though.

brad-decker committed:
remove proper noun Ethereum from all locales for onlyAddTrustedNetworks

brad-decker commented:
In the end I removed a handful of translations that just didn't seem correct or to match the original intent of the message, and kept a few. I removed the Korean translation as it included the word Ethereum in it, so we'll need a new translation for that locale. The same goes for the simplified Chinese.

I removed Japanese and Filipino translations because the google translate seemed off to me and I want to ensure we get accurate translations.

Gudahtt reviewed (approved):
LGTM!
